provisions/roles/cleanup/jenkins/master/tasks/main.yml

```diff
@@ -2,3 +2,12 @@
 - name: Stop jenkins systemd service
   sudo: yes
   service: name=jenkins state=stopped
+
+- name: Remove the virtual environment directories
+  sudo: yes
+  file: path=/opt/cccp-service state=absent
+
+- name: Remove images with no name or tag
+  sudo: yes
+  ignore_errors: yes
+  shell: "docker rmi -f $(docker images | awk '/<none>/ {print $3}')"
```

Comment:
IMHO, it's better to do `docker rmi $(docker images -q -f dangling=true)` and let Docker daemon decide which images are safe to delete.